### PR TITLE
Adds example `cli command`

### PR DESCRIPTION
**Pull request by Tomdye**
Addresses: https://github.com/dojo/cli/issues/27

Creates an example command.
`dojo example command`

### CODE REVIEW

File: dojo-cli-example/package.json [hunk 1]
```diff
@@ -22,11 +22,9 @@
     "prepublish": "grunt peerDepInstall",
     "test": "grunt test"
   },
-  "peerDependencies": {
-    "dojo-cli": "2.0.0-alpha.2"
-  },
   "devDependencies": {
     "codecov.io": "0.1.6",
+	"dojo-cli": "2.0.0-alpha.2",
```
Comment (agubler) [inline dojo-cli-example/package.json]: tab vs space

**Comment by agubler** [comment]
can you put an entry in the projects `travis.yml` for the directory so we can build it and test it?

File: dojo-cli-example/package.json [hunk 2]
```diff
@@ -22,12 +22,9 @@
     "prepublish": "grunt peerDepInstall",
     "test": "grunt test"
   },
-  "peerDependencies": {
-    "dojo-cli": "2.0.0-alpha.2"
-  },
   "devDependencies": {
     "codecov.io": "0.1.6",
-    "dojo-loader": ">=2.0.0-beta",
+	"dojo-loader": ">=2.0.0-beta",
```
Comment (matt-gadd) [inline dojo-cli-example/package.json]: formatting

**Comment by Tomdye** [comment]
Added full test coverage

**Comment by Tomdye** [comment]
@agubler added travis stuff here: https://github.com/dojo/examples/pull/38/files

**Pull request merged**